lib/modules/datasource/docker/index.ts

```diff
@@ -712,6 +713,14 @@ export class DockerDatasource extends Datasource {
     namespace: 'datasource-docker-tags',
     key: (registryHost: string, dockerRepository: string) =>
       `${registryHost}:${dockerRepository}`,
+    cacheable: (registryHost: string, _dockerRepository: string) => {
+      const { password, token } = hostRules.find({
+        hostType: dockerDatasourceId,
+        url: registryHost,
+      });
+      const secret = token ?? password;
+      return !secret;
```

Comment:
Unfortunately.. we have a token for all Docker Hub, to increase API limits. So I think this code would mean we don't cache Docker Hub tag lookups anymore.

Reply:
This function never fetches docker hub

Reply:
In that case, this could be OK